src/stores/notifications/RoomNotificationState.ts

```diff
@@ -76,6 +87,12 @@ export class RoomNotificationState extends NotificationState implements IDestroy
         }
     };
 
+    private handleRoomAccountDataUpdate = (ev: MatrixEvent) => {
```

Comment:
this doesn't feel like the right type for account data: there's a ton of missing fields on account data events.

Reply:
With every occurrence in `matrix-react-sdk` there is `.on("Room.accountData")` used along with `MatrixEvent` as the type for its first argument.

What could be the alternative type? :)